src/libc.rs

```diff
@@ -0,0 +1,7 @@
+pub fn log(x: f64) -> f64 {
+    x.ln()
+}
+
+pub fn sqrt(x: f64) -> f64 {
+    x.sqrt()
+}
```

Comment:
The only drawback to this is that it takes a bit more compilation time since the compiler will hopefully remove this indirection again. However, it makes copy-pasting C code easier so I think it makes sense? @storopoli what do you think?